src/components/SelectInput.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useRef, useState } from 'react'
+import PropTypes from 'prop-types'
+import { makeStyles } from '@material-ui/core/styles'
+import ActionMenu, {
+  ActionMenuItem,
+  ActionMenuRadio
+} from 'cozy-ui/transpiled/react/ActionMenu'
+import DropdownText from 'cozy-ui/transpiled/react/DropdownText'
+import InputGroup from 'cozy-ui/transpiled/react/InputGroup'
+import useBreakpoints from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
+
+const useStyles = makeStyles(() => ({
+  mobileInput: {
+    width: '100%',
+    maxWidth: 'none',
+    border: '0',
+    '&:hover, &:focus, &:active': {
+      border: '0'
+    }
+  },
+  desktopInput: {
+    height: '2.5rem',
+    borderRadius: '1.25rem',
+    flex: '0'
+  },
+  mobileButton: {
+    boxSizing: 'border-box',
+    padding: '.625rem .5rem',
+    lineHeight: '1.25rem',
+    whiteSpace: 'nowrap',
+    '& span, & p': {
+      display: 'inline'
+    }
+  },
+  desktopButton: {
+    boxSizing: 'border-box',
+    margin: '-.063rem',
+    padding: '.625rem 1rem',
+    borderRadius: '1.25rem',
+    lineHeight: '1.25rem',
+    whiteSpace: 'nowrap',
+    '& span, & p': {
+      display: 'inline'
+    }
+  }
+}))
+
+const SelectInput = ({ options, name, value, setValue }) => {
+  const { isMobile } = useBreakpoints()
+  const styles = useStyles()
+  const [open, setOpen] = useState(false)
```

Comment:
when the variable is a boolean, let's used "isSomething" syntax to make  it clear. I would use `isOpened` here